src/SSW.SophieBot/SSWSophieBot/dialogs/GetPeopleOnClientWorkDialog/language-generation/en-us/GetPeopleOnClientWorkDialog.en-us.lg

```diff
@@ -27,54 +28,80 @@
             "actions": [
                 {
                     "type": "Action.Submit",
-                    "title": "Show All [${employees.count}]",
+                    "title": "Show All [${count(employees)}]",
                     "data": {
                         "intent": "GetPeopleOnClientWork",
+                        "available": "${dialog.availableEntity}",
+                        "internal": "${dialog.internalEntity}",
+                        "location": "${location}",
+                        "datetime": [
+                            {
+                            "timex": [
+                                "${dialog.fromDate}"
+                            ],
+                            "type": "date"
+                            }
+                        ]
                     }
                 },
-                ${join(select(subArray(locations, 0, 5), location => LocationButton(skill, location, employees)), ',')}
+                ${join(select(subArray(locations, 0, count(locations) <= 5 ? count(locations) : 5), location => LocationButton(location, Number(location, employees))), ',')}
             ]
         },
         {
             "type": "ActionSet",
             "actions": [
-                ${join(select(subArray(locations, 5, count(locations)), location => LocationButton(skill, location, employees)), ',')}
+                ${join(select(if(subArray(locations, 5, count(locations)) != null, subArray(locations, 5, count(locations)), createArray()), location => LocationButton(location, Number(location, employees))), ',')}
             ]
         }
     ]
 }
 ```
 
-# TimeInfo(date)
-- ${if(date != null, TimeString(date), '')}
+# NameList(firstNames, peopleCount)
+- IF: ${ peopleCount <= 4 }
+    - ${firstNames}
+- ELSE:
+    - ${firstNames} and **${peopleCount - 3} others**
+
+# StatusInfo()
+- IF: ${ dialog.onInternal }
+    - are on internal work
+- ELSEIF: ${ dialog.booked }
+    - are on client work
+-ELSE:
+    - are free
 
-# TimeString(date)
+# DateInfo(date)
+- ${if(date != null, DateText(date), '')}
+
+# DateText(date)
 -``` on ${date}```
 
-# LocationButton(skill, location, employees)
+# Number(location, employees)
+- ${count(where(employees, employee => employee.defaultSite.name == location))}
+
+# LocationButton(location, number)
 -``` 
 {
     "type": "Action.Submit",
-    "title": "${location} [${Count(location, employees)}]",
+    "title": "${location} [${number}]",
     "data": {
         "intent": "GetPeopleOnClientWork",
-        "location": "${location}"
+        "available": "${dialog.availableEntity}",
+        "internal": "${dialog.internalEntity}",
+        "location": "${location}",
+        "datetime": [
+            {
+            "timex": [
+                "${dialog.fromDate}"
+            ],
+            "type": "date"
+            }
+        ]

```

Comment:
```suggestion
        ],
        "msteams": {
            "type": "messageBack",
	    "displayText": "${location} [${number}]"
        }

```

Reply:
The number after the location is not necessary. Will remove it.